index.html

```diff
@@ -0,0 +1,1104 @@
+          fill="#1E1B1B"
+        />
+      </svg>
+      <ul class="navbar__links">
+        <li class="navbar__item">
+          <a href="./index.html" class="navbar__link">Home</a>
+        </li>
+        <li class="navbar__item">
+          <a href="./pages/products.html" class="navbar__link">Product</a>
+        </li>
+        <li class="navbar__item">
+          <a href="./pages/pricing.html" class="navbar__link">Pricing</a>
+        </li>
+        <li class="navbar__item">
+          <a href="./pages/aboutUs.html" class="navbar__link">About Us</a>
+        </li>
+        <li class="navbar__item">
+          <a href="./pages/blog.html" class="navbar__link">Blog</a>
+        </li>
+        <li class="navbar__item">
+          <a href="./pages/contact.html" class="navbar__link">Contact</a>
+        </li>
+        <li class="navbar__item">
+          <button class="navbar__button">Free trial</button>
+        </li>
+      </ul>
+    </header>
+    <main>
+      <section class="header header--primary">
+        <div class="header__content">
+          <h1 class="header__headline header__headline--primary">
+            The Best Software to Grow <br class="header__linebreak" />
+            your Sales and Services
+          </h1>
+          <h5 class="header__sub-headline header__sub-headline--primary">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor <br class="header__linebreak" />
+            invidunt ut labore et dolore magna aliquyam erat.
+          </h5>
+          <form class="header__form">
+            <div class="header__form-container form__container--primary">
```

Comment:
This is not valid in BEM. You are using a different element's (supposedly `form__container` element) modifier with a `header__form-container` element.